spec/test_app/db/schema.rb

```diff
@@ -46,6 +46,7 @@
     t.datetime "updated_at"
     t.integer  "group_id"
     t.string   "banner_url"
+    t.text     "sermon_notes"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.